resource/reactivedrop_german.txt

```diff
@@ -743,8 +743,8 @@
 "[english]asw_weapon_flashlight"		"Flashlight"
 "asw_weaponl_flashlight"		"Taktisches Licht"
 "[english]asw_weaponl_flashlight"		"Flashlight Attachment"
-		"asw_weapon_flashlight_attributes"		"Tactical"
-		"[english]asw_weapon_flashlight_attributes"		"Tactical"
+"asw_weapon_flashlight_attributes"		"Passiv"
```

Comment:
Taktisch?

Reply:
Yea, but tactical doesn't really mean anything in this context. Passive tells the user something tactical about the item.

Reply:
It's a reference to https://youtu.be/urMHvdLz0Jg?t=246

Reply:
![image](https://user-images.githubusercontent.com/69652463/202896599-71ac190b-85f2-4aff-b0e9-1c4639eaed65.png)

Reply:
Ah, you moved the reference to a different string.

All good.